docs/ci.md

```diff
@@ -0,0 +1,14 @@
+# Continuous integration
+
+Kubevirt uses prow as its CI
+See [Project-infra](https://github.com/kubevirt/project-infra)
+
+## CI developer mode
+
+In order to save resources and run selectible lanes for debugging purposes,
```

Comment:
In order to save resources and run only selected lanes (i.e. for debugging) a developer can use KUBEVIRT_LANE_FOCUS environment variable (located at automation/test.sh).
The variable can be set to a list of selected lane names. The other lanes will fail immediately, thus saving CI resources and allow developers to get the results faster as well. In addition,  the PR won't merge until this selection will be reverted (unset the KUBEVIRT_LANE_FOCUS) and all the lanes will run and succeed.

Reply:
done, thanks